enterprise/app/usage/usage.tsx

```diff
@@ -182,52 +190,254 @@ export default class UsageComponent extends React.Component<UsageProps, State> {
         </div>
         <div className="card usage-card">
           <div className="content">
-            <TrendsChartComponent
-              title="Total cache download (bytes)"
-              standaloneChart={true}
-              data={dates}
-              dataSeries={[
-                {
-                  name: "total cache download (bytes)",
-                  extractValue: (ts) => +(this.getUsage(ts).totalDownloadSizeBytes ?? 0),
-                  formatHoverValue: (value) => bytes(value || 0) + " downloaded",
-                  onClick: this.onBarClicked.bind(this, "cas"),
-                  color: ChartColor.BLUE,
-                },
-              ]}
-              primaryYAxis={{
-                formatTickValue: bytes,
-                allowDecimals: false,
-              }}
-              formatXAxisLabel={(ts) => moment.unix(ts).format("MMM D")}
-              formatHoverXAxisLabel={(ts) => moment.unix(ts).format("dddd, MMMM Do YYYY")}
-              ticks={[]}
-            />
+            {detailed && (
+              <TrendsChartComponent
+                title="Total cache download (bytes)"
+                standaloneChart={true}
+                data={dates}
+                dataSeries={[
+                  {
+                    name: "internal",
+                    extractValue: (ts) => +(this.getUsage(ts).totalInternalDownloadSizeBytes ?? 0),
+                    formatHoverValue: (value) => bytes(value || 0) + " intern downloaded",
```

Comment:
is there a reason to shorten these?

Reply:
ooooops, nope, forgot to update them.  thanks!